spec/paperclip/attachment_spec.rb

```diff
@@ -1477,6 +1478,53 @@ def call(filename)
     end
   end
 
+  context "an attachment with process_in_background set" do
+    before do
+      ActiveJob::Base.queue_adapter = :test
+      ActiveJob::Base.logger = nil
+
+      rebuild_model styles: { thumb: "100x100" },
+                    only_process: [:none],
+                    process_in_background: [:thumb]
+      @file = File.new(fixture_file("5k.png"), "rb")
+      GlobalID.app = "paperclip-test"
```

Comment:
I couldn't make it work without it. In normal Rails app, this is set up by railties